controllers/namespace_controller.go

```diff
@@ -92,6 +100,27 @@ func (r *NamespaceReconciler) propagateMeta(ctx context.Context, ns, parent *cor
 			ns.Annotations[k] = v
 		}
 	}
+
+	subNS := &accuratev1.SubNamespace{}
+	err := r.Get(ctx, types.NamespacedName{Name: ns.Name, Namespace: parent.Name}, subNS)
```

Comment:
At the [line 347](https://github.com/cybozu-go/accurate/blob/1302f9f8eb9414b408fb34f466223b8649b94420/controllers/namespace_controller.go#L347), this method is called for a template instance namespace, not a sub-namespace.

I think this should look up a SubNamespace only when `ns` is a sub-namespace (i.e., it has `accurate.cybozu.com/parent` label).

Reply:
fixed.